packages/storybook/stories/Telephone.stories.jsx

```diff
@@ -55,7 +53,7 @@ const Page = () => (
     <Subtitle />
     <Description />
     <Primary />
-    <ArgsTable />
+    <ArgsTable story={PRIMARY_STORY}/>
```

Comment:
This change is due to a storybook version bump, and I based it on [current documentation](https://storybook.js.org/docs/react/writing-docs/docs-page#remixing-docspage-using-doc-blocks). Without it I wasn't seeing the args table.

![image](https://user-images.githubusercontent.com/2008881/137988002-d1819e04-88dc-40e1-9532-59d60d7c8b57.png)